### DISCUSSION TITLE
How to Ignore meta using docsify

### DISCUSSION THREAD

**Discussion by skumaravelan** [docsifyjs/docsify#1676] [Q&A] [answered]
Hi 
I am trying to ignore the below as i need these elements for git pages to order the git pages. I am not sure, how to ignore these labels from rendering using docsify. 

---
layout: default
title: "Getting Started"
nav_order: 1 
---

**Comment by trusktr** [comment, marked as answer]
Hello! Not sure what you mean. Got example?

**Reply by skumaravelan** [reply]
When you add the these meta tags in any Markdown, Docsify renders as a text in the page. I don't want to display them. How do  hide these meta tags.

**Reply by trusktr** [reply]
That is not standard markdown. You need a plugin for that. Either remove that content, or try the front matter plugin:

https://github.com/docsifyjs/docsify/issues/168
